published/006-2016-March-7.md

```diff
@@ -0,0 +1,180 @@
+# IPFS Weekly #6
+
+[IPFS](https://ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened in the past week. For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](https://github.com/ipfs) or join us on [IRC](https://webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](https://tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the [first week of March](https://github.com/ipfs/pm/issues/93):
+
+## Updates
+
+#### [**station**](https://github.com/ipfs/station)
+
+![image](https://cloud.githubusercontent.com/assets/1211152/13579734/e5dd5c96-e496-11e5-99cf-f3c09943f2f4.png)
+
+The new version of `station` is ready for developer preview! `station` is one of the easiest ways to get an IPFS daemon running on your machine. It acts as a service and you get many convenient features, such as the ability to turn an IPFS node on through a GUI and drag and drop to share through IPFS. To try it out, you need Node.js 4 and npm 3 installed. Then, do the following:
```

Comment:
suggestion: make the first "station" here a link. the image breaks up, and the link is hard to see.